ci-operator/step-registry/redhat-chaos/pod-scenarios/etcd-disruption/redhat-chaos-pod-scenarios-etcd-disruption-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: redhat-chaos-pod-scenarios-etcd-disruption
+  from_image:
+    namespace: chaos
+    name: krkn
+    tag: latest
+  commands: redhat-chaos-pod-scenarios-etcd-disruption-commands.sh
+  env:
+  - name: NAMESPACE
```

Comment:
guessing you'll have to edit this as well like pr: https://github.com/openshift/release/pull/37408/files

Reply:
Made the changes. PTAL.